### ISSUE TITLE
How can I turn off the hover magnifying effect?

### ISSUE THREAD

**Issue by middlesister** [vasturiano/timelines-chart#64]
This might be a js newbie question, but I'd like to remove the hover magnifying effect.

How can I set disableHover=true from my own script on the timlines-chart object?

**Comment by SZBoatwright** [comment]
I don't think this library supports turning off specific animations, but you can turn all animations off with `yourTimeline.enableAnimations(false)`

**Comment by eleinwang** [comment]
When I set yourTimeline.enableAnimations(false) , the magnifying effect still works , how i can only disable magnifying ?

**Comment by middlesister** [comment]
Well not really. `enableAnimations()` will disable the animations between zooming in and out. It does nothing to affect the magnifying effect on hover, sadly.

**Comment by chancesmith** [comment]
I'd like to add this feature. 🙋‍♂️
